app/models/preapplication_service.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class PreapplicationService < ApplicationRecord
```

Comment:
Wonder whether this should be more generic like `AdditionalService` so that it's reusable

Reply:
Perhaps we can create an `AdditionalService` and turn `PreapplicationService` into a subclass of it if it turns out we need them in future?

Reply:
It's just weird as a data model to have an association that's specific to a subset of that model - if we use STI and had `class PreApplication < PlanningApplication` then I'd be less bothered if it was defined in the subclass.

Leave it if you want but what will happen if we need it for other services in the future is we'll end up with non-preapplication services in that table but the model will still be called `PreapplicationService` because MVP trumps everything - either that or we'll add another model. 😭

Reply:
Oh yeah, I forgot that Rails wants you to design everything up front.

Reply:
Lets just leave it as `PreapplicationService` and if it needs changing I'll make sure it happens

Reply:
Ignore the last comment - I see you've changed it now